flopy/utils/reference.py

```diff
@@ -1858,7 +1852,7 @@ def from_argus_export(cls, fname, nlay=1):
         return cls(xc, yc, verts, iverts, np.array(nlay * [len(iverts)]))
 
     def __setattr__(self, key, value):
-        super(SpatialReference, self).__setattr__(key, value)
+        super().__setattr__(key, value)
```

Comment:
whoops, this is method is part of `SpatialReferenceUnstructured` (which is a subclass of `SpatialReference`), so this single edit should be undone.